etc/profile-a-l/joplin.profile

```diff
@@ -0,0 +1,71 @@
+# Firejail profile for joplin
+# Description: A note taking and to-do application
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include joplin.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+include allow-bin-sh.inc
+include allow-nodejs.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+include whitelist-common.inc
+
+mkdir ${HOME}/.config/joplin
+whitelist ${HOME}/.config/joplin
+
+whitelist ${HOME}/.config/nano
+read-only ${HOME}/.config/nano
+whitelist ${HOME}/.emacs
+read-only ${HOME}/.emacs
+whitelist ${HOME}/.emacs.d
+read-only ${HOME}/.emacs.d
+whitelist ${HOME}/.nanorc
+read-only ${HOME}/.nanorc
+whitelist ${HOME}/.vim
+read-only ${HOME}/.vim
+whitelist ${HOME}/.viminfo
+read-only ${HOME}/.viminfo
+whitelist ${HOME}/.vimrc
```

Comment:
Move the whitelist in the whitelist block and the read-only between the private and the dbus block.